src/angular/table/table.md

```diff
@@ -477,6 +477,36 @@ The flag affects the sequence at runtime, not at definition time.
 </ng-container>
 ```
 
+### Table Cell Actions
+
+It is possible to define up to three action buttons (or links) for one row.
+The buttons are displayed either when hovering or when navigating with the keyboard.
+See the example "Cell Actions incl. Keyboard Navigation Support" to see how keyboard
+navigation can be implemented.
+Please be careful when using these action buttons because the user will
+not immediately see that there are actions.
+We recommend for accessibility reasons to always create a click action on the whole row,
+which links to a possible detail page, where all actions can be called again.
+Also, the action buttons should always be provided with aria labels,
+so that screen readers can output a meaningful text.
+
+```html
+<td>
+  <div class="sbb-table-cell-actions" aria-label="Press tab key to access actions.">
+    <a sbb-icon-button href="#" attr.aria-label="Edit {{element.vehicle}}.">
```

Comment:
```suggestion
    <a sbb-icon-button [routerLink]="['./edit', element.id]" attr.aria-label="Edit {{element.vehicle}}.">
```

Reply:
But we leave it in the example, or what do you think?